### PR TITLE
PP-661: replace circleci with gh actions

### PR DESCRIPTION
**Pull request by antomor**
## What

- Replace CircleCI with Github actions

## Why

- Because CircleCi fails when using github dependencies.

### CODE REVIEW

**Review by david-iov — APPROVED** [review]
LGTM, you probably want to also remove this repo from [Circle CI](https://app.circleci.com/pipelines/github/rsksmart/rif-relay/1839)

**Comment by antomor** [comment]
@david-iov thank you for reviewing it.
> LGTM, you probably want to also remove this repo from [Circle CI](https://app.circleci.com/pipelines/github/rsksmart/rif-relay/1839)
Not yet, because the main branch still runs on circle ci, but we'll do it after merging develop into main.

File: package.json [hunk 1]
```diff
@@ -64,13 +64,13 @@
...
-    "clean": "rm -rf typechain-types cache bin dist artifacts contracts-exposed",
-    "postclean": "hardhat clean",
+    "prebuild": "hardhat clean",
+    "build": "hardhat compile --force",
     "format": "prettier --list-different  --no-editorconfig \"{,test/**/}*.ts\"",
     "format:fix": "npm run format -- --write",
     "lint": "npx eslint --cache \"{,test/**/}*.ts\"",
     "lint:fix": "npm run lint -- --fix",
-    "prepare": "utils/prepare",
+    "prepare": "utils/prepare.npm",
```
Comment (jurajpiar) [inline package.json:73]: new payments team standard? I like it.

**Review by jurajpiar — APPROVED** [review]
I trust you that it works ... ;] Just make sure you have trailing new lines everywhere and that you remove the CircleCi pipeline from jobs.

File: package.json (hunk 1 above)
Comment (antomor) [reply package.json:73]: I like your idea, indeed. It clarifies its purpose.

**Comment by antomor** [comment]
> Just make sure you have trailing new lines everywhere
Good point, thank you!

**Pull request merged**